scripts/Muon/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-include (UiToPy)
+include(UiToPy)
```

Comment:
As we don't have any `.ui` files to compile we shouldn't need this file at all.

Reply:
I guess because there's nothing in `set(UI_FILES)`, there won't be anything compiled so I'm free to make the changes?

Reply:
Going ahead because it builds fine and FDA works as expected